Templates/SendGrid-CSharp/function.json

```diff
@@ -10,6 +10,7 @@
             "type": "sendGrid",
             "name": "message",
             "direction": "out",
+            "apiKey":  "SendGridApiKey",
             "from": "Azure Functions <samples@functions.com>",
```

Comment:
Note: I removed them in my latest commit. If they're needed, I can put them back, but I don't think they are.